express-api/src/controllers/notifications/notificationsController.ts

```diff
@@ -48,3 +50,49 @@ export const getNotificationsByProjectId = async (req: Request, res: Response) =
     return res.status(500).send({ message: 'Error fetching notifications' });
   }
 };
+
+export const resendNotificationById = async (req: Request, res: Response) => {
+  const id = Number(req.params.id);
+  const notification = await notificationServices.getNotificationById(id);
+  if (!notification) {
+    return res.status(404).send('Notification not found.');
+  }
+  const kcUser = req.user;
+  if (!(isAdmin(kcUser) || isAuditor(kcUser))) {
```

Comment:
If they are not an admin, should we even allow them to trigger this?

Reply:
No idea, I wasn't sure how strict this should be so I just applied similar logic as elsewhere.